corehq/pillows/application.py

```diff
@@ -35,16 +35,18 @@ def get_app_to_elasticsearch_pillow(pillow_id='ApplicationToElasticsearchPillow'
         doc_prep_fn=transform_app_for_es
     )
     change_feed = KafkaChangeFeed(
-        topics=[topics.APP], client_id='apps-to-es', num_processes=num_processes, process_num=process_num
+        topics=[topics.APP], client_id='apps-to-es', num_processes=num_processes, process_num=process_num, dedicated_migration_process=dedicated_migration_process,
```

Comment:
nit: stickler is not going to like the long line

Reply:
Noted Danny. thanks